### PR TITLE
Support configuring the logger with `options.service`, etc.

### PR DESCRIPTION
**Pull request by rmharrison**
If this PR fixes a bug, you _must_ add test cases representative of the bug.

#### What does this PR do?
Previously `options.service` test was commented because passing options to the logger wasn't yet supported.
```
# test/format.test.js
// expect(parsedTestMess.service).toEqual(options.service);
```

#### How should this be manually tested?
Follow README.md instructions and see that the options output to the logger
```
> logger.info('message', {foo: 'bar', baz: 'qux'});
{"service":"test-service","logger":"Winston-JSON-Formatter","hostname":"<>","level":"info","msg":"message","meta":{"service":{"version":"1.0.0","node_env":""},"logger":{"time":"2018-11-28T02:52:06.700Z"},"event":{"foo":"bar","baz":"qux"}},"err":{}}
```

#### Screenshots (if appropriate):

### CODE REVIEW

File: test/format.test.js [hunk 1]
```diff
@@ -14,86 +14,100 @@ const options = {
...
             done();
         });
     });
-    describe('json output', () => {
+
+    describe('json output (info)', () => {
         beforeEach(() => {
-            // spy to check output
             spy = jest.fn((info) => {
+                expect(spy).toHaveBeenCalled();
```
Comment (morleegan) [inline test/format.test.js]: `testLoggerInfo` is also calling this check. Remove from here and leave in `testLoggerInfo`

**Review by morleegan — APPROVED** [review]
LGTM

**Comment by morleegan** [comment]
@rmharrison - leaving for you to merge

**Comment by rmharrison** [comment]
Holding-off on merging pending getting CI setup.

**Comment by rmharrison** [comment]
"Do not merge" removed because Jenkins CI is now setup.
Thanks @nickmccally

**Comment by rmharrison** [comment]
@puckybreg : Let me know when you've had a look, then I'll merge and cut release `0.9.0` (manual `npm publish` for the time being).

**Review by puckybreg — APPROVED** [review]
👍

**Pull request merged**